src/content/en/tools/workbox/guides/_shared/generate-sw-string-schema.html

```diff
@@ -0,0 +1,87 @@
+<tr>
+  <td>
+    <p>globDirectory</p>
```

Comment:
You could use devsite includes to share the definitions like this one.

Create `_shared/configurations/globDirectory.html` and just add:
```
<td>
    ....
</td>
```

and in appropriate places add: 
```
<tr>
    {% include '/web/tools/workbox/_shared/configuration/globDirectory.html' %}
    <td>
        ....
    </td>
</tr>
```

Reply:
Now notifications that this is different - but I am guessing generate-sw-string is for webpack since it's mentioning self.__precache etc.

Reply:
Yes, this is one of the places where there's a different default value, so it can't be shared.